ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -952,7 +955,10 @@ const ControlledTierCard = (props: {
               is required.
             </div>
             <div style={styles.requestAccess}>
-              <SupportButton label='Request Access' />
+              <SupportButton
```

Comment:
I like how concise this is.  But I'm not sure whether this use case should live inside SupportButton - it seems to me like it's pretty specific to D-A-R.  What do you think?

Reply:
Good point. I did notice that SupportButton is only used in D-A-R. I am good with moving it.